Extractor/extractor.cpp

```diff
@@ -1,37 +1,96 @@
 #include "extractor.h"
+#include "IsoHandler.h"
 #include <argparse.hpp>
 #include <filesystem>
-#include "IsoHandler.h"
 
-void Extractor(const char* iso, const char* output)
+#include "Zero2/Zero2Extractor.h"
+
+void InvalidISOError()
 {
-  auto extractor = IsoHandler::Extractor(iso, output);
+  std::cerr << "Failed to parse ISO file." << std::endl;
 
-  extractor.Extract();
+  std::cerr << "If this is a valid Zero/Fatal Frame/Project Zero ISO file"
+            << std::endl
+            << "Then please either open a report on the github here:"
+            << std::endl
+            << "https ://github.com/wagrenier/Mikompilation" << std::endl
+            << std::endl
+            << "or contact us on our Discord Server: " << std::endl
+            << "https://discord.gg/Ap4Sfcmwd9" << std::endl;
 }
 
+// Just added this for debugging without needing to pass arguments
+constexpr int DEBUG_MODE = 0;
+
 int main(int argc, char *argv[])
 {
-  argparse::ArgumentParser program("Mikompilation Extractor");
+  std::filesystem::path isoFile;
+  std::filesystem::path outputDirectory;
+
+  if (DEBUG_MODE)
```

Comment:
No need of that you should just be using your IDE correctly instead of hardcoding values.